sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/coverage/CxxCoverageSensor.java

```diff
@@ -179,7 +179,7 @@ private void saveMeasures(SensorContext context,
           }            
         }
       } else {
-        CxxUtils.LOG.warn("Cannot find the file '{}', ignoring coverage measures", filePath);
+        CxxUtils.LOG.trace("Cannot find the file '{}', ignoring coverage measures", filePath);
```

Comment:
What is the command line option to turn trace on?
Up to now we I think we have no trace. Maybe better debug?